### PR TITLE
Feature/username on sale

### PR DESCRIPTION
**Pull request by lucca65**
- [x] Adjust user filter to look like Figma's design
- [x] Create User module for account/username and avatar rendering
- [x] Minor font and color adjustments for the main menu
- [x] Reposition main menu links (shop now the second element)
- [x] Remove the ability to use the shop/shop viewer as a guest (it wasn't being used)
- [x] Show sale creator information on list (both mobile and desktop)
- [x] Show sale creator information on sale viewer
- [x] Remove containers on both shop and dashboard

### CODE REVIEW

File: src/styles/legacy_main.css [hunk 1]
```diff
@@ -3130,18 +3130,6 @@ textarea.form-input {
     line-height: 14px;
 }
 
-
```
Comment (zacck) [inline src/styles/legacy_main.css]: 👍

File: src/elm/Page/Transfer.elm [hunk 2]
```diff
@@ -194,34 +195,14 @@ viewTransferCard loggedIn transfer state =
 
                 Transferred ->
                     transfer.to
+
+        viewUser_ =
+            User.view loggedIn.shared.endpoints.ipfs loggedIn.accountName loggedIn.shared.translations
```
Comment (zacck) [inline src/elm/Page/Transfer.elm]: +1  for the abstraction

File: src/elm/Transfer.elm [hunk 3]
```diff
@@ -47,18 +48,15 @@ type alias CommunityArgs =
 
 type alias Transfer =
     { id : Int
-    , to : TransferUser
-    , from : TransferUser
+    , to : User
```
Comment (zacck) [inline src/elm/Transfer.elm]: +1

**Comment by Pancakem** [comment]
> @lucca65 awesome work I like the modifications it makes for easier to read and re use code. @Pancakem this would be a nice one for you to read through

@zacck thanks. This is real nice man. I'm taking away a lot. Thanks guys.

**Pull request merged**